pegasus/sites.v3/code.org/public/courses.haml

```diff
@@ -0,0 +1,101 @@
+---
+title: Courses
+theme: responsive
+---
+
+- (1..13).each do |n|
+  :css
+    .courses .bar-#{n} {
+      float: left;
+      width: #{n * 7}%;
+    }
+
+:css
```

Comment:
You know how I feel about css being in a separate file. ^_^ I'd be ok with it if it was in a course_table.haml view and that's used in several educate pages.